From 460dc0405a3a7b05c4ebfbed53f47a0fbd09304b Mon Sep 17 00:00:00 2001
From: Sandy Huang <hjc@rock-chips.com>
Date: Wed, 3 Jul 2019 16:27:33 +0800
Subject: [PATCH] arm64: dts: rockchip: rk3399: delete vop iommu PD

iommu driver probe will register vop pd to power domain control, this pd
will be closed before display driver enable and lead to display black.
so we move vop pd to display driver control.

Change-Id: Ie25c20e79c57c4e1bc678774ad09b60fc4a135da
Signed-off-by: Sandy Huang <hjc@rock-chips.com>
---
 arch/arm64/boot/dts/rockchip/rk3399.dtsi | 2 --
 1 file changed, 2 deletions(-)

diff --git a/arch/arm64/boot/dts/rockchip/rk3399.dtsi b/arch/arm64/boot/dts/rockchip/rk3399.dtsi
index acf413d01f72..24524dc36e1f 100644
--- a/arch/arm64/boot/dts/rockchip/rk3399.dtsi
+++ b/arch/arm64/boot/dts/rockchip/rk3399.dtsi
@@ -1809,7 +1809,6 @@
 		interrupt-names = "vopl_mmu";
 		clocks = <&cru ACLK_VOP1>, <&cru HCLK_VOP1>;
 		clock-names = "aclk", "iface";
-		power-domains = <&power RK3399_PD_VOPL>;
 		#iommu-cells = <0>;
 		status = "disabled";
 	};
@@ -1878,7 +1877,6 @@
 		interrupt-names = "vopb_mmu";
 		clocks = <&cru ACLK_VOP0>, <&cru HCLK_VOP0>;
 		clock-names = "aclk", "iface";
-		power-domains = <&power RK3399_PD_VOPB>;
 		#iommu-cells = <0>;
 		status = "disabled";
 	};
-- 
2.35.3

